From: Andreas Gruenbacher <agruen@suse.de>
Subject: genksyms: fix the __attribute__ rule

Gcc __attribute__ definitions may occur repeatedly, e.g.,

	static int foo __attribute__((__used__))
		       __attribute__((aligned (16)));

Fix the genksyms parser to understand this.

Signed-off-by: Andreas Gruenbacher <agruen@suse.de>

---
 scripts/genksyms/parse.y |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/scripts/genksyms/parse.y
===================================================================
--- a/scripts/genksyms/parse.y
+++ b/scripts/genksyms/parse.y
@@ -446,7 +446,7 @@ member_bitfield_declarator:
 
 attribute_opt:
 	/* empty */					{ $$ = NULL; }
-	| ATTRIBUTE_PHRASE
+	| attribute_opt ATTRIBUTE_PHRASE
 	;
 
 asm_definition:
